src/util/TicketValidator.java

```diff
@@ -0,0 +1,87 @@
+package util;
+
+import model.BusTicket;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class TicketValidator {
+    Map<String, Integer> statisticsErrors = new HashMap<>();
+
+    public String getMostFrequentError() {
+        if (statisticsErrors.isEmpty()) {
+            return "No errors recorded";
+        }
+        String mostFrequentError = null;
+        int maxCount = 0;
+        for (Map.Entry<String, Integer> entry : statisticsErrors.entrySet()) {
+            String error = entry.getKey();
+            int count = entry.getValue();
+            if (count > maxCount) {
+                mostFrequentError = error;
+                maxCount = count;
+            }
+        }
+
+        return mostFrequentError;
+    }
+
+    public boolean validateBusTicket(BusTicket busTicket) {
```

Comment:
can you devide this big method to smallers?

Reply:
Sure! Thanks)